src/components/Light/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from "react";
+
+import './style.css';
+
+import lightOff from './images/light-off.svg';
+import lightOn from './images/light-on.svg';
+
+const Light = ({name, state}) => {
+    const [rozsviceno, setRozsviceno] = useState(state);
+
+    return (
+        <div className="light" onClick={() => {(rozsviceno === 'on') ? setRozsviceno('off') : setRozsviceno('on')}}> 
```

Comment:
na 2 stavové věci se místo stringu (on/off, open/closed, ...) používá většinou boolean (true/false). Tj kdybys chtěla dá se to ještě zjednodušit na

```
const [rozsviceno, setRozsviceno] = useState(state === "on");

return (
        <div className="light" onClick={() => {setRozsviceno(!rozsviceno)}> 
```
protože boolean můžeš jenodušše znegovat :))